sickbeard/server/web/core/base.py

```diff
@@ -91,6 +91,10 @@ def __init__(self, rh, filename):
             'newsBadge': '',
             'toolsBadge': '',
             'toolsBadgeClass': '',
+            'base_url': urljoin(rh.request.protocol + '://' + rh.request.host,
+                                sickbeard.WEB_ROOT) +
+                          ('', '/')
+                          [urljoin(rh.request.protocol + '://' + rh.request.host, sickbeard.WEB_ROOT)[-1] not in '/'],
```

Comment:
Yeah sure. If I change that, we still need to test this some more. But, then are we confident that we can use this?